varnish-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,936 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
```

Comment:
Would rename this file to 'varnish-overview.libsonnet' to match the dashboard name